### PR TITLE
Half baked task prerequisites support

### PR DESCRIPTION
**Pull request by VictorLowther**
Not fully ready yet:

- No unit tests verifying that the Right Thing will happen when modifying a task list from a running task.  I suspect there are corner cases to deal with there.
- No unit tests for messing with the trask list without a Stage or Workflow.  That is mainly due to laziness.

### CODE REVIEW

File: backend/task.go [hunk 1]
```diff
@@ -9,6 +10,11 @@ import (
 	"github.com/digitalrebar/store"
 )
 
+var (
```
Comment (galthaus) [inline backend/task.go]: delete these
